library/kani_macros/src/sysroot/contracts/shared.rs

```diff
@@ -66,9 +66,6 @@ impl<'a> ContractConditionsHandler<'a> {
         // We should consider a better strategy than just duplicating all attributes.
         #[cfg(feature = "no_core")]
         {
-            self.output.extend(quote!(
-                #[unstable(feature="kani", issue="none")]
-            ));
```

Comment:
The comment above needs to be updated.

Reply:
What's the failure you are getting?

Reply:
I see failures with nearly all function-contract related regression tests (71 failed). The exact failure depends on the type of function contract attribute used, but the underlying error seems to be the same.

For example, this is one of the failures for an `ensures` example. 

```
---- [expected] expected/function-contract/checking_from_external_mod.rs stdout ----

error: test failed: expected output to contain the line(s):
- Status: SUCCESS
- Description: "|result : &u32| (*result == x) | (*result == y)"
in function max
status: exit status: 1
command: "kani" "/home/ubuntu/kani/tests/expected/function-contract/checking_from_external_mod.rs" "-Zfunction-contracts"
stdout:
------------------------------------------
Kani Rust Verifier 0.53.0 (standalone)
error: /home/ubuntu/kani/target/kani/bin/kani-compiler exited with status exit status: 101

------------------------------------------
stderr:
------------------------------------------
error[E0734]: stability attributes may not be used outside of the standard library
 --> /home/ubuntu/kani/tests/expected/function-contract/checking_from_external_mod.rs:5:1
  |
5 | #[kani::ensures(|result : &u32| (*result == x) | (*result == y))]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `kani::ensures` (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0734`.

------------------------------------------
```

and another example, 

```
------------------------------------------


---- [expected] expected/function-contract/arbitrary_requires_pass.rs stdout ----

error: test failed: expected output to contain the line(s):
arithmetic_overflow
- Status: SUCCESS
- Description: "attempt to divide by zero"
in function div
status: exit status: 1
command: "kani" "/home/ubuntu/kani/tests/expected/function-contract/arbitrary_requires_pass.rs" "-Zfunction-contracts"
stdout:
------------------------------------------
Kani Rust Verifier 0.53.0 (standalone)
error: /home/ubuntu/kani/target/kani/bin/kani-compiler exited with status exit status: 101

------------------------------------------
stderr:
------------------------------------------
error[E0734]: stability attributes may not be used outside of the standard library
 --> /home/ubuntu/kani/tests/expected/function-contract/arbitrary_requires_pass.rs:5:1
  |
5 | #[kani::requires(divisor != 0)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `kani::requires` (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0734`.

------------------------------------------
```

Removing that blob that injects the unstable attribute, fixes all 71 errors at once.

Reply:
Yes, I think this is worth a bit more investigation. As you mentioned, this should not be triggered here. It's possible that the `unstable` proc_macro is broken too.

Reply:
This part is now fixed. There is no need to remove this unstable injection, and the final result is working as expected. This error was caused by this addition to cargo.toml which made no_core the default feature for kani.
Resolving the comment now.